PukingPredator/Assets/Scripts/Item.cs

```diff
@@ -21,7 +33,45 @@ public void MoveItem(Vector3 position)
     public void Initialize(GameObject newPrefab, Vector3 newPosition, Quaternion newRotation)
     {
         prefab = newPrefab;
+        initialScale = prefab.transform.localScale;
+        rb = prefab.GetComponent<Rigidbody>();
+
         position = newPosition;
         rotation = newRotation == default ? Quaternion.identity : newRotation;
     }
+
+    public void Collect(Vector3 pos)
+    {
+        playerPos = pos;
+        collect = true;
+    }
+
+    public void Update()
+    {
+        if (collect)
```

Comment:
can we rename collect to `playingCollectionAnimation` or something, because that seems to be the utility of this variable

Reply:
Called it isCollecting since we can expand this to having more logic associated to it that just the animation